components/dropdown/README.md

```diff
@@ -21,190 +29,230 @@
 
+<!-- docs: start hidden content -->
 ![screenshot the basic dropdown-more opener](./screenshots/dropdown-more.png)
+<!-- docs: end hidden content -->
 
+<!-- docs: demo live name:d2l-dropdown-more autoSize:false align:flex-start size:medium -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/dropdown/dropdown-more.js';
   import '@brightspace-ui/core/components/dropdown/dropdown-content.js';
 </script>
-
+<script>
+  window.addEventListener('load', function () {
+    var dropdown = document.querySelector('d2l-dropdown-more');
+	setTimeout(() => {
+		dropdown.toggleOpen();
+	}, 100);
+  });
+</script>
 <d2l-dropdown-more text="Open!">
   <d2l-dropdown-content>
     Some content...
   </d2l-dropdown-content>
 </d2l-dropdown-more>
 ```
 
-**Properties:**
+<!-- docs: start hidden content -->
+### Properties
 
 | Property | Type | Description |
-|--|--|--|
+|---|---|---|
 | `disabled` | Boolean, default: `false` | Disables the dropdown opener |
 | `no-auto-open` | Boolean, default: `false` | Prevents the dropdown from opening automatically on click or key press |
 | `text` | String | Used as `aria-label` for the button |
 | `translucent` | Boolean, default: `false` | Attribute for busy/rich backgrounds |
 | `visible-on-ancestor` | Boolean, default: `false` | See [visible-on-ancestor-mixin](../../mixins/visible-on-ancestor-mixin.md) for more details on configuring that behavior |
+<!-- docs: end hidden content -->
 
-**Accessibility:**
+### Accessibility
 
-To make your usage of `d2l-dropdown-more` accessible, use the following property:
+To make your usage of `d2l-dropdown-more` accessible, use the following property
 
 | Attribute | Description |
-|--|--|
+|---|---|
 | `text` | Acts as a primary label and tooltip and is **REQUIRED**. |
 
-## Dropdown Contents
-
-### d2l-dropdown-content
-
+## Content: Container [d2l-dropdown-content]
 `d2l-dropdown-content` is a generic container for dropdown content.  It provides behavior such as sizing,  positioning, and managing focus gain/loss.
 
+<!-- docs: start hidden content -->
 ![screenshot the basic dropdown-content](./screenshots/dropdown-content.png)
 ![screenshot the dropdown-content with mobile-tray="right"](./screenshots/dropdown-content-mobile-tray.png)
+<!-- docs: end hidden content -->
 
+<!-- docs: demo live name:d2l-dropdown-content autoSize:false align:flex-start size:medium -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/dropdown/dropdown-button.js';
   import '@brightspace-ui/core/components/dropdown/dropdown-content.js';
 </script>
-
+<script>
+  window.addEventListener('load', function () {
+    var dropdown = document.querySelector('d2l-dropdown-button');
+	setTimeout(() => {
+		dropdown.toggleOpen();
+	}, 100);
+  });
+</script>
 <d2l-dropdown-button text="Open!" primary>
   <d2l-dropdown-content min-width="150" max-width="400">
```

Comment:
Do we want to add a mobile tray (ie. mobile-tray="bottom") to display the trays somewhere on this page? I was thinking to maybe add one of each tray as a demo, but putting it in with this example should be fine too. Not sure what the default size is for the snippets but it would activate at 616px.

Reply:
I added it as default enabled within the live demo, it won't show until you resize the demos but once you hit `616px` it does appear